src/main/resources/benchmarks/auctionmark/ddl-oracle.sql

```diff
@@ -54,42 +55,42 @@ CREATE TABLE REGION (
 -- u_r_id           User's region ID
 -- ================================================================
 CREATE TABLE USERACCT (
-    u_id                INT NOT NULL,
-    u_rating            INT NOT NULL,
+    u_id                VARCHAR2(128) NOT NULL,
+    u_rating            NUMBER(19, 0) NOT NULL,
     u_balance           FLOAT NOT NULL,
     u_comments			INTEGER DEFAULT 0,
-    u_r_id              INT NOT NULL REFERENCES REGION (r_id),
+    u_r_id              NUMBER(19, 0) NOT NULL REFERENCES REGION (r_id),
     u_created           TIMESTAMP,
     u_updated           TIMESTAMP,
-    u_sattr0            VARCHAR(64),
-    u_sattr1            VARCHAR(64),
-    u_sattr2            VARCHAR(64),
-    u_sattr3            VARCHAR(64),
-    u_sattr4            VARCHAR(64),
-    u_sattr5            VARCHAR(64),
-    u_sattr6            VARCHAR(64),
-    u_sattr7            VARCHAR(64),
-    u_iattr0            INT DEFAULT NULL,
-    u_iattr1            INT DEFAULT NULL,
-    u_iattr2			INT DEFAULT NULL,
-    u_iattr3            INT DEFAULT NULL,
-    u_iattr4            INT DEFAULT NULL,
-    u_iattr5            INT DEFAULT NULL,
-    u_iattr6            INT DEFAULT NULL,
-    u_iattr7            INT DEFAULT NULL, 
+    u_sattr0            VARCHAR2(64),
+    u_sattr1            VARCHAR2(64),
+    u_sattr2            VARCHAR2(64),
+    u_sattr3            VARCHAR2(64),
+    u_sattr4            VARCHAR2(64),
+    u_sattr5            VARCHAR2(64),
+    u_sattr6            VARCHAR2(64),
+    u_sattr7            VARCHAR2(64),
+    u_iattr0            NUMBER(19, 0) DEFAULT NULL,
+    u_iattr1            NUMBER(19, 0) DEFAULT NULL,
+    u_iattr2			NUMBER(19, 0) DEFAULT NULL,
```

Comment:
```suggestion
    u_iattr2            NUMBER(19, 0) DEFAULT NULL,
```
nit: align spacing if you're going to rewrite the file anyways